config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -0,0 +1,114 @@
+dex:
+  enabled: true
+  hubHostName: neurohackademy.2i2c.cloud
+
+staticWebsite:
+  enabled: true
+  source:
+    git:
+      repo: https://github.com/NeuroHackademy-2023/zoom/
+      branch: main
+  ingress:
+    host: neurohackademy.2i2c.cloud
+    path: /zoom
+  githubAuth:
+    enabled: true
+
+jupyterhub:
+  ingress:
+    hosts:
+      - neurohackademy.2i2c.cloud
+    tls:
+      - secretName: https-auto-tls
+        hosts:
+          - neurohackademy.2i2c.cloud
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
+        org:
+          name: NeuroHackademy
+          url: https://neurohackademy.org/
+          logo_url: https://user-images.githubusercontent.com/118582/178122521-aa5b83e7-f898-415a-89ff-dc288f599b4f.png
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: The National Institutes of Health grant 2R25MH112480-06
+          url: https://reporter.nih.gov/search/ydTvTwXxk0yd6eGdRznbLQ/project-details/10409452
+  singleuser:
+    # User image: https://quay.io/repository/arokem/nh2023?tab=tags
+    image:
+      name: quay.io/arokem/nh2023
+      tag: "9aecdbb2881f"
+    nodeSelector:
+      2i2c.org/community: neurohackademy
+    extraTolerations:
+      - key: "2i2c.org/community"
+        operator: "Equal"
+        value: "neurohackademy"
+        effect: "NoSchedule"
+    cpu:
+      guarantee: 0.5
+      # We're on n1-highmem-16 machines
+      limit: 14
+    memory:
+      guarantee: 4G
+      limit: 8G
+  hub:
+    config:
+      JupyterHub:
+        authenticator_class: cilogon
+      Authenticator:
+        allowed_users: &neurohackademy_users
+          - arokem
+        admin_users: *neurohackademy_users
+      CILogonOAuthenticator:
+        scope:
+          - "profile"
+        oauth_callback_url: https://neurohackademy.2i2c.cloud/hub/oauth_callback
+        shown_idps:
+          - https://github.com/login/oauth/authorize
+        allowed_idps:
+          http://github.com/login/oauth/authorize:
+            username_derivation:
+              username_claim: "preferred_username"
+    extraFiles:
+      configurator-schema-default:
+        data:
+          properties:
+            Spawner.default_url:
+              type: string
+              title: Default User Interface
+              enum:
+                - "/tree"
+                - "/lab"
+                - "/git-pull?repo=https%3A%2F%2Fgithub.com%2FNeuroHackademy-2023%2Fcurriculum&urlpath=lab&branch=main"
+              default: "/git-pull?repo=https%3A%2F%2Fgithub.com%2FNeuroHackademy-2023%2Fcurriculum&urlpath=lab&branch=main"
```

Comment:
IIUC, this line should make the interface lab by default and also execute the nbgitpuller link. However, this does not seem to be the case. Locally, I have tried setting this value to `/lab` and redeploying, and I'm still redirected to the notebook interface. So I don't really know whats happening here.

Reply:
When I visit the configurator, its configured to `/lab` and not the git pull one. I figure you just got to switch to `git-pull there.

![image](https://github.com/2i2c-org/infrastructure/assets/3837114/6b3bd824-04c5-4ce2-9f61-5a29debfd716)

Reply:
Maybe that was a separate thing to consider, but thinking about this, I wonder if the `urlpath=` is as it should be.... Yes it looks alright without including `/`..

Not sure.

Reply:
I got lab to load when I set it in the configurator (overriding the git-pull url in the config)

Reply:
Then when I went back to the configurator and selected the git-pull URL, everything worked as expected. So I think there might be something up with the initial config populating the default, but once you use the configurator to override, it's all fine 🤷🏻‍♀️